dependencies/common/.gitignore

```diff
@@ -17,5 +18,6 @@ profvis/
 profvis*.tar.gz
 *.tar.bz
 *.tar.bz2
+*.tar.gz
```

Comment:
should this be `r2d3*.tar.gz`?

Reply:
I don't believe there's a use for keeping *.tar.gz from being ignored, is there? If you agree, I'll leave this and remove the other .tar.gz listings here.

Reply:
Yes, I can't see why we'd want to commit a .tar.gz file here! Let's do as you suggest and remove the other `.tar.gz` entries.